### ISSUE TITLE
Clean-up branches

### ISSUE THREAD

**Issue by arthurmloureiro** [LSSTDESC/augur#66]
There are just tooooo many branches atm. As we develop more Augur, we should try to trim some of the branches :)

**Comment by fjaviersanchez** [comment]
Right! I just took a look and it's not as trivial as I thought it would be. I'll need to follow up with a few folks.

**Comment by arthurmloureiro** [comment]
Oh no need to rush!
